bliss/api.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
I believe we need this because of executing `main` from this file as `bliss` executable?

Reply:
Still works fine for me because there's a shebang in the file named `bliss` that poetry generates:
```
#!/home/regier/bliss/.venv/bin/python
import sys
from bliss.api import main

if __name__ == '__main__':
    sys.exit(main())
```